vector/agent/namespaced/daemonset.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: apps/v1
```

Comment:
Is there a source for this - or did you compose this yourself?

Reply:
no, I copied the manifests we deploy already, for example: https://github.com/utilitywarehouse/kubernetes-manifests/blob/master/exp-1-aws/sys-log/vector.yaml and took it from there. I will see if I can find an upstream and wrap it in a make target

Reply:
https://github.com/vectordotdev/vector/tree/master/distribution/kubernetes there are base manifests here, but from that I strip down a few mount paths we are not using and also change the image to a debian one that comes with journalctl. Given how simple this deployment looks I am not sure this worth following any upstream

Reply:
Thats fine, I was wondering about labels:

```
    app.kubernetes.io/name: vector
    app.kubernetes.io/instance: vector
    app.kubernetes.io/component: Agent
```

`instance` and `component` have limited value and looked like they came from upstream I didn't think you would add them.